rules/opentitan.bzl

```diff
@@ -145,10 +146,13 @@ elf_to_disassembly = rule(
     incompatible_use_toolchain_transition = True,
 )
 
-def _elf_to_scrambled(ctx):
+def _elf_to_scrambled_rom_impl(ctx):
     outputs = []
     for src in ctx.files.srcs:
-        scrambled = ctx.actions.declare_file("{}.scr.40.vmem".format(src.basename))
```

Comment:
The name changed from "*.scr.40.vmem" --> "*.scr.39.vmem", because according to the [rom_ctrl documentation](https://docs.opentitan.org/hw/ip/rom_ctrl/doc/) for each ROM word there are 32 bits of data and 7 ECC bits, for a total of 39 bits per word. This also matches the [previous naming scheme in meson](https://github.com/lowRISC/opentitan/blob/536092a26c5f5dc7f24a87c48d4849c5a2eb972d/sw/device/meson.build#L122).